guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -24,13 +19,32 @@ ifndef::orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --disable '*'
+# subscription-manager repos \
+ifdef::satellite[]
+--enable={RepoRHEL8ServerSatelliteServerProductVersion} \
+endif::[]
+--enable={RepoRHEL8BaseOS} \
+--enable={RepoRHEL8AppStream}
+----
+ifdef::katello[]
++
+. Install the `katello-repos-latest.rpm` package
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf localinstall https://yum.theforeman.org/katello/{KatelloVersion}/katello/el8/x86_64/katello-repos-latest.rpm
```

Comment:
You'll also need the foreman-release RPM. This is because the katello module depends on foreman